build/cpp/func/src/func_test.cc

```diff
@@ -54,23 +52,6 @@ TEST(Func, GetEvent) {
   EXPECT_EQ(reply.followers()[0], "username_0");
 }
 
-// test invalid event
-//  uses warble request with invalid parent for testing
-TEST(Func, Invalid) {
-  func::Func func;
-  func.SetFuncMap(warble_functions::WarbleFunctions::GetFuncMap());
-  func.Hook(0, "Warble");
-  WarbleRequest request;
-  WarbleReply reply;
-  Any any;
-  request.set_username("username_0");
-  request.set_text("Text for user_0");
-  request.set_parent_id("0");
-  any.PackFrom(request);
-  const auto &optional_any = func.Event(0, any);
-  EXPECT_EQ(optional_any, std::nullopt);
-}
-
```

Comment:
Invalid parents are no longer relevant as ids are randomly generated at the command line level.